diff --git a/node_modules/@shopify/react-native-skia/lib/module/Platform/index.js b/node_modules/@shopify/react-native-skia/lib/module/Platform/index.js
index ddf7476..2011e1e 100644
--- a/node_modules/@shopify/react-native-skia/lib/module/Platform/index.js
+++ b/node_modules/@shopify/react-native-skia/lib/module/Platform/index.js
@@ -1,2 +1,2 @@
-export * from "./Platform";
+export * from "./Platform.js";
 //# sourceMappingURL=index.js.map
\ No newline at end of file
diff --git a/node_modules/@shopify/react-native-skia/lib/module/index.js b/node_modules/@shopify/react-native-skia/lib/module/index.js
index 5b2ca00..f016ba3 100644
--- a/node_modules/@shopify/react-native-skia/lib/module/index.js
+++ b/node_modules/@shopify/react-native-skia/lib/module/index.js
@@ -1,4 +1,4 @@
-import "./skia/NativeSetup";
+import "./skia/NativeSetup.js";
 export { JsiSkImage } from "./skia/web/JsiSkImage";
 export * from "./renderer";
 export * from "./renderer/Canvas";
diff --git a/node_modules/@shopify/react-native-skia/lib/module/skia/NativeSetup.js b/node_modules/@shopify/react-native-skia/lib/module/skia/NativeSetup.js
index 1c480da..c54eec5 100644
--- a/node_modules/@shopify/react-native-skia/lib/module/skia/NativeSetup.js
+++ b/node_modules/@shopify/react-native-skia/lib/module/skia/NativeSetup.js
@@ -1,5 +1,5 @@
-import { Platform } from "../Platform";
-import NativeSkiaModule from "../specs/NativeSkiaModule";
+import { Platform } from "../Platform/index.js";
+import NativeSkiaModule from "../specs/NativeSkiaModule.js";
 if (Platform.OS !== "web" && global.SkiaApi == null) {
   // Initialize RN Skia
   const SkiaModule = NativeSkiaModule;
